search/controllers/advanced/__init__.py

```diff
@@ -132,6 +132,9 @@ def _query_from_form(form: forms.AdvancedSearchForm) -> AdvancedQuery:
     q = _update_query_with_dates(q, form.date.data)
     q = _update_query_with_terms(q, form.terms.data)
     q = _update_query_with_classification(q, form.classification.data)
+    include_older_versions = form.include_older_versions.data
+    if include_older_versions:
+        q.include_older_versions = include_older_versions 
```

Comment:
If you're not using the local ``include_older_versions`` elsewhere, you may not need it:

```python
if form.include_older_versions.data:
    q.include_older_versions = True
```
or
```python
q.include_older_versions = True if form.include_older_versions.data else False
```

Reply:
The latter may be overlong, however

Reply:
Thanks, went with first option